src/sanctoral/a.yml

```diff
@@ -0,0 +1,555 @@
+  sex: female
+  titles:
+    - Virgin
+
+aidan:
+  canonisationLevel: Saint
+  firstName: Aidan
+  sex: male
+  titles:
+    - Bishop
+
+aidanAndTheSaintsOfLindisfarne:
+  join:
+    - aidan
+    - saintsOfLindisfarne
+
+aidanOfLindisfarne:
+  canonisationLevel: Saint
+  firstName: Aidan
+  surname: of Lindisfarne
+  sex: male
+  titles:
+    - Bishop
+    - Missionary
+
+ailbe:
+  canonisationLevel: Saint
+  firstName: Ailbe
+  sex: male
+  titles:
+    - Bishop
+
+alban:
+  canonisationLevel: Saint
+  firstName: Alban
+  sex: male
+  titles:
+    - Martyr
+
+albanAndJuliusAndAaron:
+  join:
+    - alban
+    - julius
+    - aaron
+
+albertChmielowski:
+  canonisationLevel: Saint
+  firstName: Albert
+  surname: Chmielowski
+  sex: male
+  titles:
+    - Religious
+
+albertinaBerkenbrock:
+  canonisationLevel: Blessed
+  firstName: Albertina
+  surname: Berkenbrock
+  sex: female
+  titles:
+    - Virgin
+    - Martyr
+
+albertoHurtado:
+  canonisationLevel: Saint
+  firstName: Alberto
+  surname: Hurtado
+  sex: male
+  titles:
+    - Priest
+
+albertTheGreat:
+  canonisationLevel: Saint
+  firstName: Albert
+  surname: the Great
+  sex: male
+  titles:
+    - Bishop
+    - Doctor
+    - Abbot
+
+alfonsoRodriguezOlmedo:
+  canonisationLevel: Saint
+  firstName: Alfonso
+  surname: Rodriguez Olmedo
+  sex: male
+  titles:
+    - Priest
+    - Martyr
+
+allSaints:
+  canonisationLevel: Saint
+  firstName: All Saints
+  number: 1000
+
+allSaintsOfIreland:
+  canonisationLevel: Saint
+  firstName: All Saints
+  surname: of Ireland
+  number: 1000
+
+allSaintsOfWales:
+  canonisationLevel: Saint
+  firstName: All Saints
+  surname: of Wales
+  number: 1000
+
+allSouls:
+  canonisationLevel: Both
+  firstName: All Souls
+  number: 1000
+
+aloysiusGonzaga:
+  canonisationLevel: Saint
+  firstName: Aloysius
+  surname: Gonzaga
+  sex: male
+  titles:
+    - Religious
+
+aloysiusStepinac:
+  canonisationLevel: Saint
+  firstName: Aloysius
+  surname: Stepinac
+  sex: male
+  titles:
+    - Bishop
+    - Martyr
+
+alphonsaOfTheImmaculateConception:
+  canonisationLevel: Saint
+  firstName: Alphonsa
+  surname: of the Immaculate Conception
+  sex: female
+  titles:
+    - Virgin
+
+alponsusMariaDeLiguori:
+  canonisationLevel: Saint
+  firstName: Alponsus
+  additionalFirstName: Maria
+  surname: de’ Liguori
+  sex: male
+  titles:
+    - Bishop
+    - Doctor
+
+amand:
+  canonisationLevel: Saint
+  firstName: Amand
+  sex: male
+  titles:
+    - Missionary
+
+ambrose:
+  canonisationLevel: Saint
+  firstName: Ambrose
+  sex: male
+  titles:
+    - Bishop
+    - Doctor
+
+ambroseFrancisFerro:
+  canonisationLevel: Blessed
+  firstName: Ambrose
+  additionalFirstName: Francis
+  surname: Ferro
+  sex: male
+  titles:
+    - Priest
+    - Martyr
+  notes: Ambrosio Francisco Ferro
+
+anastasius:
+  canonisationLevel: Saint
+  firstName: Anastasius
+  sex: male
+  titles:
+    - Martyr
+  celebrationDate1969: 22/01
+  birthDate: 6th century
```

Comment:
I don't know yet what is the best format. But we should find one that will be easy to catch and limit the risk or errors.
Anyway a test could still be made to check if all these dates are valid.

Reply:
Take your time to think about it.

And another notation should be created for approximate date, for example:
- `a [any notation from above]` (e.g. `approx. 1710` → `a1710`.

Although, it might be easier / more readable to use `approx` (or `approx.`).

I don’t about the syntax/notation, but I think the _definers_ should be able to input any of the above-mentioned dates.